.dockerignore

```diff
@@ -0,0 +1,3 @@
+*
+!docker/
+!pkg/docker
```

Comment:
Ah, gotcha!  I must have misunderstood the syntax;  that does clarify why `*` is the first entry....

I'll fix.